ui/src/components/flags/FlagTable.tsx

```diff
@@ -228,12 +227,12 @@ export default function FlagTable(props: FlagTableProps) {
       </table>
       {table.getPageCount() > 1 && (
         <Pagination
-          className="mt-4"
           currentPage={table.getState().pagination.pageIndex + 1}
```

Comment:
the pagination also seems to disapear whenever they set it to anything over the current number of items. for example if there are 21 items and they set it to 50, there is no way to change it to anything else.

.gif attached

![2023-12-06 17 25 28](https://github.com/flipt-io/flipt/assets/209477/47225a0e-9bcc-411f-a6d0-df59652664d6)

Reply:
I updated it so it will stay on the page if there is more items than the lowest pagesize.